### PR TITLE
HomeMatic CCU add-on retirement/migration path

### PR DESCRIPTION
**Pull request by jens-maus**
As discussed in https://github.com/home-assistant/operating-system/pull/1683, the old "HomeMatic CCU" add-on in this repository is going to be retired/flagged as being obsolete in favor of the much more advanced third-party [RaspberryMatic CCU](https://github.com/jens-maus/RaspberryMatic/tree/master/home-assistant-addon). See also the [RaspberryMatic HomeAssistant Add-on Documentation](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant).

Therefore, this PR implements the following changes:

- removes the `subsystem=tty` device filter type allowing to specify an arbitrary device node for connecting the `HmIP-RFUSB` device which will require a different non-tty device node `/dev/raw-uart` to be used with an upcoming HAos update. This should allow users (who cannot immediately migrate to the new add-on) to continue using the old-style add-on.
- adds a `createBackup.sh` script and a patched version of the `cp_security.cgi` WebUI file which both are internally used to generate CCU compatible `*.sbk` system backup files which will contain all main configuration files required. These `*.sbk` backup files can then be restored within RaspberryMatic so that existing HomeMatic CCU add-on users can more easily migrate to the new add-on
- adds some warnings to the major README, CHANGELOG and DOCS markdown files to get still existing users of the old homematic add-on notified about the fact that the old homematic add-on is dropped in favor of the new RaspberryMatic CCU add-on.

### CODE REVIEW

File: homematic/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+#### :warning: This add-on is considered to be obsolete/retired in favor of the much more advanced third-party [RaspberryMatic CCU](https://github.com/jens-maus/RaspberryMatic/tree/master/home-assistant-addon) add-on for running a HomeMatic/homematicIP smart home central within HomeAssistant. If you want to migrate to the new add-on, please make sure to update to the latest version of this old "HomeMatic CCU" add-on first and then use the WebUI-based backup routines to export a `*.sbk` config backup file which you can then restore in the new "RaspberryMatic CCU" add-on afterwards (cf. [RaspberryMatic Documentation](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant)) :warning:
+
+## 11.4.0
```
Comment (pvizeli) [inline homematic/CHANGELOG.md:5]:
```suggestion
## 99.0.0
```

File: homematic/config.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: 11.3.0
+version: 11.4.0
```
Comment (pvizeli) [inline homematic/config.yaml:1]:
```suggestion
version: 99.0.0
```

**Review by pvizeli — APPROVED** [review]
LGTM. Thanks @jens-maus

File: homematic/CHANGELOG.md (hunk 1 above)
Comment (jens-maus) [reply homematic/CHANGELOG.md:5]: @pvizeli Why are you suggesting to set the version number to 99.0.0 ?

Comment (pvizeli) [reply homematic/CHANGELOG.md:5]: Is okay now?

Comment (jens-maus) [reply homematic/CHANGELOG.md:5]: it's fine, yes.

**Pull request merged**